### PR TITLE
Fix Conditional flow tests

### PR DESCRIPTION
**Pull request by mastepan**
//test: `@integrations-conditional-flows`

### CODE REVIEW

**Comment by syndesisqe** [comment]
Checkstyle verification failed with:
```
ui-tests/src/test/java/io/syndesis/qe/pages/integrations/editor/add/steps/
  ConditionalFlows.java:26:31: '{' is not followed by whitespace. [WhitespaceAround]
ui-tests/src/test/java/io/syndesis/qe/pages/integrations/editor/add/steps/
  ConditionalFlows.java:26:32: '}' is not preceded with whitespace. [WhitespaceAround]
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: ui-tests/src/test/java/io/syndesis/qe/pages/integrations/editor/add/steps/ConditionalFlows.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class ConditionalFlows {
+
+    private static final class Elements {
+        //temporary xpath, change to css datatestid if present - should be in the next build
```
Comment (asmigala) [inline ui-tests/src/test/java/io/syndesis/qe/pages/integrations/editor/add/steps/ConditionalFlows.java]: I think this comment can be deleted, the selector now uses `testid`

File: ui-tests/src/test/java/io/syndesis/qe/pages/integrations/editor/add/steps/ConditionalFlows.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        public static final By ADD_ANOTHER_CONDITION = By.cssSelector("button[data-testid=\"form-array-control-add-another-item-button\"]");
+    }
+
+    @Setter
+    @Getter
+    private String templateDef;
```
Comment (asmigala) [inline ui-tests/src/test/java/io/syndesis/qe/pages/integrations/editor/add/steps/ConditionalFlows.java]: This does not seem to be used anywhere

File: ui-tests/src/test/resources/features/integrations/conditional-flow.feature [hunk 3]
```diff
@@ -223,20 +225,21 @@ Feature: Conditional flows - content base routing
 
     When fill in values by element data-testid
       | flowconditions-0-condition | ${body.message} == 'Shaco' |
-    And click on the "Done" button
-    Then check that conditional flow step contains 2 flows
-    And check that conditional flow default step is disabled
+    And click on the "Next" button
+    And click on the "Next" button
+    Then check that conditional flow step contains 1 flows
```
Comment (asmigala) [inline ui-tests/src/test/resources/features/integrations/conditional-flow.feature]: Is it expected that there are now fewer flows than before?

File: ui-tests/src/test/java/io/syndesis/qe/pages/integrations/editor/add/steps/ConditionalFlows.java (hunk 1 above)
Comment (mastepan) [reply ui-tests/src/test/java/io/syndesis/qe/pages/integrations/editor/add/steps/ConditionalFlows.java]: fixed, thanks

File: ui-tests/src/test/java/io/syndesis/qe/pages/integrations/editor/add/steps/ConditionalFlows.java (hunk 2 above)
Comment (mastepan) [reply ui-tests/src/test/java/io/syndesis/qe/pages/integrations/editor/add/steps/ConditionalFlows.java]: fixed, thanks

File: ui-tests/src/test/resources/features/integrations/conditional-flow.feature (hunk 3 above)
Comment (mastepan) [reply ui-tests/src/test/resources/features/integrations/conditional-flow.feature]: yes, this is intentional

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**